fileio/ft_filetype.m

```diff
@@ -1539,6 +1540,11 @@
   type = 'duneuro_dgf';
   manufacturer = 'duneuro';
   content = 'geometrical meshes';
+  % known York Instruments file types
+elseif filetype_check_extension(filename, '.meghdf5')
+  type = 'York_Instruments_hdf5';
```

Comment:
would you mind changing this into `yorkinstruments_hdf5`, i.e. a single string for the first part and only lower case? That nomenclature will make things easier for future extensions, such as adding a template layout for plotting (see https://www.fieldtriptoolbox.org/template/layout/)

Reply:
That seems sensible, I have changed all instances in latest commit.